app/_templates/page.njk

```diff
@@ -12,7 +47,7 @@
   {{ header({
     "showNav": "false",
     "showSearch": "false",
-    "homeHref": "https://www.nhs.uk"
+    "homeHref": "/"
```

Comment:
This won't work due to the GitHub pages being hosted at `/nhsuk-frontend/` using `/` would return you to `https://nhsuk.github.io/` - which gives an error page. We had this problem in the past, which is why we used the nhs.uk website URL. Using `/nhsuk-frontend` would fix it on the GitHub pages but this wouldn't work locally as we work at the root directory.

Reply:
Replaced `/` with `/nhsuk-frontend/`